src/mbgl/util/dtoa.cpp

```diff
@@ -0,0 +1,97 @@
+#include "dtoa.hpp"
+
+#include <rapidjson/internal/dtoa.h>
```

Comment:
It's internal, but RapidJSON is header only, so we're not going to run into ABI-incompatibilities, and we'll notice as soon as they change the internal function signatures (which is unlikely).